lib/linguist/languages.yml

```diff
@@ -3369,6 +3369,8 @@ Perl:
   - ".psgi"
   - ".t"
   filenames:
+  - Makefile.PL
+  - Rexfile
```

Comment:
In-the-wild usage is a little questionable... only [~155 results](https://github.com/search?q=filename%3ARexfile+NOT+nothack&type=Code), although they appear to be well-distributed across users/repositories (which technically meets the criteria of "being used in hundreds of repositories"). I think it's okay, particularly since it's an uncommon filename unlikely to cause conflict. @pchaigno, thoughts?

Reply:
I'll defer to @lildude :smile:

Reply:
Problem here is that Rexfile was actually in the Perl6 field (my fault: https://github.com/github/linguist/pull/2005/files). It should better be here, Rexfile is in no way Perl6 code.

Reply:
This is fine. It slipped under the radar before when it was initially added so no point removing it now.